src/Controls/src/Core/TapGestureRecognizer.cs

```diff
@@ -42,17 +46,22 @@ public int NumberOfTapsRequired
 			set { SetValue(NumberOfTapsRequiredProperty, value); }
 		}
 
-		public event EventHandler Tapped;
+		public ButtonsMask Buttons
+		{
+			get { return (ButtonsMask)GetValue(ButtonsProperty); }
+			set { SetValue(ButtonsProperty, value); }
+		}
+
+		public event EventHandler<TappedEventArgs>? Tapped;
 
-		internal void SendTapped(View sender)
+		internal void SendTapped(View sender, Func<IElement?, Point?>? getPosition)
```

Comment:
Can `getPosition` default to `null` and reduce the noise of all the `SendTapped(..., null)` calls?

Reply:
👍 

Also @rookiejava FYI if you want to extend your usages of "SendTapped" to provide location data.